File: Java/Debug/src/Seq.java
import java.util.Arrays;

public class Seq {

    private int start;
    private int increment = 1;
    private int count;

    public Seq(int start, int increment, int count) {
        this.start = start;
        this.increment = increment;
        this.count = count;
    }

    public String toString() {
        return Arrays.toString(this.getSeq());
    };

    public int[] getSeq() {
        int cnt = 0;
        int[] L = new int[this.count];
        int x = this.start;
        while (cnt < this.count) {
            L[cnt] = x;
            x += this.increment;
            cnt += 1;
        }
        return L;
    }
}

File: Java/Debug/seq.py
class seq:
    
    __start =  None
    __increment = 1
    __count = None
    
    def __init__(self,start,increment,count):
        self.__start = start
        self.__increment = increment
        self.__count = count
        
    def __str__(self):
        return str(self.getSeq())
        
    def getSeq(self):
        cnt = 0
        L = []
        x = self.__start
        while cnt < self.__count:
            L.append(x)
            x += self.__increment
            cnt += 1
        return L
            